index.html

```diff
@@ -0,0 +1,75 @@
+<!DOCTYPE html>
+<html lang="en">
+    <head>
+        <meta charset="utf-8">
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Roland Portfolio</title>
+        <link rel="stylesheet" href="https://fonts.googleapis.com/css?family=Poppins">
+        <link rel="stylesheet" href="css/style.css">
+    </head>
+
+    <body>
+        <div class="wrapper" id="wrapper">
+            <header class="header">
+                <nav class="nav">
+                    <img src="img/image_geometry_1.svg" alt="">
+                    <a href="#" onclick="showMenu()"><img class="menu-button" src="img/ic_menu.svg" alt="Menu Buttom"></a>
+                </nav>
+            </header>
+
+            <section class="headline">
+                <div class="title set">
+                    <h1>Roland<br> Ossisa Yuma</h1>
+                </div>
+                <div class="content set">
```

Comment:
- [ ] Well done on adding the first section of the portfolio but kindle stick as much as you can to [Figma design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=23%3A9). Keep rocking 💪 

Here you have a screenshot from your project:

![image](https://user-images.githubusercontent.com/74506933/158685072-b556925f-93f5-4531-b66c-6cc0ffb7091a.png)

Here you have a screenshot from the design:

![image](https://user-images.githubusercontent.com/74506933/158685142-51ae9bd4-643f-4ae3-a833-4f7f043a4cd7.png)